sdk/src/main/java/com/horizen/cryptolibprovider/ThresholdSignatureCircuitWithKeyRotation.java

```diff
@@ -49,7 +49,7 @@ Boolean verifyProof(List<WithdrawalRequestBox> bt,
                         byte[] proof,
                         String verificationKeyPath);
 
-    byte[] generateSysDataConstant(byte[] genesisKeysRootHash, long threshold);
+    byte[] generateSysDataConstant(List<byte[]> signerPublicKeysList, List<byte[]> masterPublicKeysList, long threshold);
```

Comment:
Agree on using lambda function but I think is more correct to use the appropriate function `SchnorrPublicKey::deserialize` instead of` BytesUtils::fromHexString`